packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/useHandleUpload.ts

```diff
@@ -71,10 +72,11 @@ export function useHandleUpload({
 }): [tasks: CancelableTask[], handleUpload: () => void] {
   const getConfig = useGetLocationConfig();
   const [tasks, setTasks] = React.useState<CancelableTask[]>(() =>
-    (items ?? []).map(({ key, data, size }) => ({
+    (items ?? []).map(({ key, data, size, type }) => ({
       cancel: () => setTasks((prev) => removeTask(prev, key)),
       key,
       size,
+      type,
```

Comment:
oh that's a good question, I think we actually assign the `type` inside `ActionSelect` as `FILE` because of the `FileItem` type

```tsx
export interface FileItem {
  key: string;
  data?: File;
  lastModified: Date;
  size: number;
  type: 'FILE';
}
```

It does look like the actual type is exposed though. Do you think it makes sense to use the real type instead of `FILE`? Screenshot below of the type I'm able to see when adding a file

![image](https://github.com/user-attachments/assets/2af913de-38e0-4c37-8e12-aacbedca364e)